### ISSUE TITLE
dismissPasscodeLock(animated: Bool = true) called many times?

### ISSUE THREAD

**Issue by klwoon** [velikanov/SwiftPasscodeLock#49]
Thanks for the great effort maintaining this library!

Please refer to the screenshot below captured after repeatedly putting the Demo app into background, then to foreground to unlock, and background again:
![screen shot 2017-05-26 at 6 33 37 pm](https://cloud.githubusercontent.com/assets/9354867/26491487/5ff8a242-4242-11e7-97ff-8a76e63c3c2e.png)

It appears that the method `dismissPasscodeLock(animated: Bool = true)` will be called many times, is that the expected behaviour? I was testing why the dismiss animation only works for the first time unlock but not the subsequent unlock, then found this.

Please advise.

**Comment by ziogaschr** [comment]
I don't think this is expected indeed. Nice catch. It will be nice if you can create a PR. Otherwise, I will try to have a look soon (not in the next 10 days)
